packages/trader/package.json

```diff
@@ -85,6 +85,7 @@
         "webpack-node-externals": "^2.5.2"
     },
     "dependencies": {
+        "@cloudflare/stream-react": "^1.9.1",
```

Comment:
do we need package-lock.json since you added a new library

Reply:
Is it a wrong package-lock.json?
<img width="858" alt="Screenshot 2023-12-22 at 11 49 14" src="https://github.com/binary-com/deriv-app/assets/121025168/dca599bd-7e1d-46dd-b1e7-340e1939bf1c">

Reply:
oh i didnt see it because it was at the very top lol